kotlin-native/NEW_MM.md

```diff
@@ -56,14 +48,16 @@ pluginManagement {
 }
 ```
 
-### Switch to the new memory model
+### Switch to the new MM
+
+Add the compilation flag `-Xbinary=memoryModel=experimental` using one of the following options:
```

Comment:
This might sound unclear to users who don't use Gradle. Also, Gradle options work on other level of abstraction, so mentioning that they enable this flag unnecessarily reveals an implementation detail.
What do you think about the following wording:

> Add the compilation flag `-Xbinary=memoryModel=experimental`. In Gradle, you can alternatively do one of the following:

?

Same for `-Xbinary=freezing=disabled`.